plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/pylint/PyLintAnalysisTest.java

```diff
@@ -44,54 +50,52 @@ public void testMarkersInfoOutput()
                         "        else:\n" +
                         "            x = letters.index(letter) - shift\n" +
                         "            encoded = encoded + letters[x]\n" +
-                        "print(encoded)");
+                        "print(encoded)\n" +
+                        "print(\"This line is really long\")" +
+                        " # This comment is part of the long line, going over the 100 char limit");
 
         PyLintAnalysis pyLintAnalysis = new PyLintAnalysis(null, document, null, new NullProgressMonitor(),
                 null);
 
-        String output = "************* Module snippet\n" +
-                "C: 14, 0: (bad-whitespace) Exactly one space required around assignment\n" +
-                "            encoded=encoded + letters[x]\n" +
-                "\n" +
-                "                   ^\n" +
-                "C: 22, 0: (missing-final-newline) Final newline missing\n" +
-                "C: 22, 0: (superfluous-parens) Unnecessary parens after 'print' keyword\n" +
-                "C:  1, 0: (missing-docstring) Missing module docstring\n" +
-                "C:  3, 0: (invalid-name) Constant name \"shift\" doesn't conform to UPPER_CASE naming style\n" +
-                "C:  4, 0: (invalid-name) Constant name \"choice\" doesn't conform to UPPER_CASE naming style\n" +
-                "C:  5, 0: (invalid-name) Constant name \"word\" doesn't conform to UPPER_CASE naming style\n" +
-                "C:  6, 0: (invalid-name) Constant name \"letters\" doesn't conform to UPPER_CASE naming style\n" +
-                "C:  7, 0: (invalid-name) Constant name \"encoded\" doesn't conform to UPPER_CASE naming style\n" +
-                "\n" +
-                "------------------------------------------------------------------\n" +
-                "\n" +
-                "Your code has been rated at 5.26/10 (previous run: 5.26/10, +0.00)\n";
+        String output = "PyLint: The stdout of the command line is:\n"
```

Comment:
The patch seems nice, but the previous code checked had a case which was removed which had multiple lines (the `bad-whitespace`), so, it'd be nice to have that case in the new code too.

Reply:
I've had a look and pylint no longer outputs that message (since version [2.6.0](http://pylint.pycqa.org/en/latest/whatsnew/changelog.html#what-s-new-in-pylint-2-6-0) )

> bad-continuation and bad-whitespace have been removed, black or another formatter can help you with this better than Pylint

I've had a look at how it handles the output from an old pylint version. It splits the output into lines, and checks whether each line matches the regex, meaning that the extra lines are ignored as they don't match the regex.

Do you think it is worth using output from the old version of PyLint?

Reply:
I think it's ok just to check that those lines are actually ignored (given that this is the only place that checks it).

Reply:
(because presumably there are other errors reported in pylint which could have multiple lines)